internal/publish/publish.go

```diff
@@ -130,14 +130,15 @@ func NewServer(ctx context.Context, cfg *Config, env *serverenv.ServerEnv) (*Ser
 	}, nil
 }
 
-func (s *Server) Routes(ctx context.Context, cfg *Config) *mux.Router {
+func (s *Server) Routes(ctx context.Context) *mux.Router {
 	logger := logging.FromContext(ctx).Named("publish")
 
 	r := mux.NewRouter()
+	r.Use(middleware.ProcessChaff(s.tracker))
```

Comment:
Should this come after maintenance mode and other middlewares?

Reply:
(It was previously first, so I kept that behavior, but I'm wondering if it's the _correct_ behavior)